### PR TITLE
Rename `simulation-service` to `sciml-service`

### PR DESCRIPTION
**Pull request by fivegrant**
This PR renames 'simulation-service' to 'sciml-sevice' in the bake and compose.

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]
Now you need to update `orchestration` for the new name.

**Comment by fivegrant** [comment]
Going to wait until Tuesday to merge this just in case this causes downstream problems. @YohannParis here's the orchestration PR: https://github.com/DARPA-ASKEM/orchestration/pull/306

**Pull request merged**
